test/test_helpers/storage.rb

```diff
@@ -34,9 +43,18 @@ def nodes
           @nodes || DEFAULT_NODES
         end
 
-        def mock_storage_client!
-          require '3scale/backend/storage_async'
-          class << ::ThreeScale::Backend::StorageAsync::Client
+        def mock_storage_clients
+          STORAGE_CLASSES.each { |klass| mock_storage_client!(klass) }
+        end
+
+        def unmock_storage_clients
+          STORAGE_CLASSES.each { |klass| unmock_storage_client!(klass) }
+        end
```

Comment:
This commit is a candidate for squashing with the one introducing the async client in tests.

Reply:
I prefer to leave this one like it is.